### PR TITLE
Add C jejuni db, some more social photos

### PR DESCRIPTION
**Pull request by johnlees**

### CODE REVIEW

File: content/poppunk/index.md [hunk 1]
```diff
@@ -54,10 +54,10 @@ We welcome contributions for new species, or expansions of these databases [via
     <tr>
     <td><i>Campylobacter jejuni</i></td>
     <td style="text-align:center">v1</td>
-    <td>42453</td>
+    <td>40303</td>
     <td style="text-align:center"><a href="https://www.ncbi.nlm.nih.gov/pathogens/">Portal</a></td>
-    <td style="text-align:center">Coming soon</td>
-    <td style="text-align:center"></td>
+    <td style="text-align:center"><a href="https://ftp.ebi.ac.uk/pub/databases/pp_dbs/Campylobacter_jejuni_v1_refs.tar.bz2">Download</a></td>
```
Comment (samhorsfield96) [inline content/poppunk/index.md:59]: When I try to download either refs or full dbs I get to a page that says I don't have permission.

Comment (johnlees) [reply content/poppunk/index.md:59]: Need to run `bsub -I -q datamover_debug chmod +r /nfs/ftp/public/databases/pp_dbs/Campylobacter*` after uploading

**Comment by samhorsfield96** [comment]
All good fixed my own error with access to C. jejuni models

**Review by samhorsfield96 — APPROVED** [review]
All good.

**Pull request merged**